runtime/altair/src/lib.rs

```diff
@@ -2659,7 +2769,7 @@ mod benches {
 		[pallet_proxy, Proxy]
 		[pallet_utility, Utility]
 		[pallet_scheduler, Scheduler]
-		[pallet_collective, Council]
+		[pallet_collective_council, Council]
```

Comment:
NOTE: We now have two benchmarks for the `pallet_collective` based on the `CouncilCollective` and `TechnicalCollective` instances.